packages/datetime/src/timePicker.tsx

```diff
@@ -105,25 +112,33 @@ export class TimePicker extends React.Component<ITimePickerProps, ITimePickerSta
         precision: TimePickerPrecision.MINUTE,
         selectAllOnFocus: false,
         showArrowButtons: false,
+        useAmPm: false,
     };
 
     public static displayName = "Blueprint2.TimePicker";
 
     public constructor(props?: ITimePickerProps, context?: any) {
         super(props, context);
 
+        let value = props.minTime;
+        let useAmPm = TimePicker.defaultProps.useAmPm;
```

Comment:
this is not necessary. `props.useAmPm` will already be set to this. you can remove this variable entirely.